### DISCUSSION TITLE
PreCompose - Kotlin 2.0.0 / Compose 1.6.x

### DISCUSSION THREAD

**Discussion by ThraaxSession** [Tlaster/PreCompose#337] [Q&A]
Hi together,

I get all time compilation errors when I migrated to Kotlin 2.0.0 and Compose 1.6.x. Now I try to figure out which dependency isn't ready.
So, should PreCompose work without any issue?

**Comment by Tlaster** [comment]
I will push an update with Kotlin 2.0 and Compose 1.6 support tomorrow, but I'm not sure current 1.5.x version works with Kotlin 2.0.
